src/PluginManager/Microsoft.Performance.Toolkit.PluginManager.Core/Packaging/Metadata/DataSourceType.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Performance.Toolkit.PluginManager.Core.Packaging.Metadata
+{
+    public enum DataSourceType
+    {
+        FileDataSource,
+        ExtensionlessFileDataSource,
```

Comment:
I'm wondering if instead of having a bunch of information about the actual supported data sources we just want something on `PluginMetadata` like

`PluginMetadata.SupportsFolders`
`PluginMetadata.SupportedExtensions`
`PluginMetadata.SupportsExtensionlessFiles`

From the user's perspective, they really only care about what can be processed by installing the plugin. Thoughts?

Reply:
Or putting those properties on `ProcessingSourceMetadata` instead of `PluginMetadat`

Reply:
Let's chat about it in call.

Reply:
Since `description` would probably be needed in the UI, I'm going to keep it as is for now.